client/data/disputes/hooks.ts

```diff
@@ -43,6 +43,27 @@ export const useDispute = (
 	return { dispute, isLoading, doAccept };
 };
 
+export const useDisputeAccept = (
```

Comment:
This hook is a wrapper for the `acceptTransactionDetailsDispute` action and an `isLoading` listener.